cyclops/models/wrappers/pt_model.py

```diff
@@ -958,7 +961,7 @@ def get_predictions(examples):
             preds_ds = dataset.map(
                 get_predictions,
                 batched=True,
-                batch_size=batch_size,
+                batch_size=self.batch_size,
```

Comment:
Have you run the use-case notebook after using the self.batch_size? I remember it would cause an error if there was some inconsistency between the batch sizes at different stages. I need to check the code again.

Reply:
I ran it once on the huggingface exploration notebook and it worked. I figured the class already had a batch size, so it was redundant to specify it specifically for prediction. This change also means that the evaluator can have a different batch size than the one used for model training/inference, which can speed things up (especially when the image column is dropped during evaluation).

Reply:
yes that makes sense. I will check on that again to make sure no error persists.